modules/apim/manifests/params.pp

```diff
@@ -36,6 +38,19 @@
   # Define remove file list
   $file_removelist = []
 
+  # ----- Master-datasources config params -----
+  $stat_db_url = 'jdbc:h2:../tmpStatDB/WSO2AM_STATS_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000;AUTO_SERVER=TRUE'
+  $stat_db_username = 'wso2carbon'
+  $stat_db_password = 'wso2carbon'
+  $stat_db_driver = 'org.h2.Driver'
+  $stat_db_validation_query = 'SELECT 1'
+
+  $mb_store_db_url = 'jdbc:h2:repository/database/WSO2MB_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000'
+  $mb_store_db_username = 'wso2carbon'
+  $mb_store_db_password = 'wso2carbon'
```

Comment:
Shall we remove this?

Reply:
Both stat db and mb store dbs.

Reply:
Removed parameters https://github.com/wso2/puppet-apim/pull/142/commits/3e4e676ed49b9087341aabd594579669aecdf306, https://github.com/wso2/puppet-apim/pull/142/commits/53f7f89d0526fc08b7636f73b7c86a4a6bacf512